Services/BookingService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Airport_Ticket_Booking_System.Models;
+using Airport_Ticket_Booking_System.Models.Enums;
+using Airport_Ticket_Booking_System.Repositories;
+
+namespace Airport_Ticket_Booking_System.Services;
+
+
+public static class BookingService
+{
+    public static void BookFlight(User user, string userAnswer)
+    {
+        if (string.IsNullOrEmpty(userAnswer))
+            throw new ArgumentException("Invalid Flight ID");
+
+        if (!int.TryParse(userAnswer, out int flightIdInt))
+            throw new ArgumentException("Invalid Flight ID");
```

Comment:
Since they have the same exceptions and the same message. 
```suggestion
        if (string.IsNullOrEmpty(userAnswer) || !int.TryParse(userAnswer, out int flightIdInt))
            throw new ArgumentException("Invalid Flight ID");
```